custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_light.yaml

```diff
@@ -8,74 +8,91 @@ card_light:
     ulm_card_light_name: "[[[ return entity.attributes.friendly_name ]]]"
     ulm_card_light_icon: "[[[ return entity.attributes.icon ]]]"
     ulm_card_light_enable_collapse: false
-    ulm_card_light_enable_slider: false
-    ulm_card_light_enable_slider_minSet: 0
-    ulm_card_light_enable_slider_maxSet: 100
     ulm_card_light_enable_horizontal: false
     ulm_card_light_enable_horizontal_wide: false
     ulm_card_light_enable_color: false
+    ulm_card_light_color_palette: ""
     ulm_card_light_force_background_color: false
+    ulm_card_light_enable_slider: false
+    ulm_card_light_enable_slider_minSet: 0
+    ulm_card_light_enable_slider_maxSet: 100
+    ulm_card_light_enable_buttons: false
+    ulm_card_light_brightness_low: 2
+    ulm_card_light_brightness_medium: 128
+    ulm_card_light_brightness_high: 255
     ulm_card_light_enable_popup: false
     ulm_card_light_enable_popup_tap: false
-    ulm_card_light_color_palette: ""
   show_icon: false
   show_name: false
   show_label: false
   state:
     - value: "on"
       styles:
         card:
-          - background-color: >-
+          - background-color: >
               [[[
                   var color = entity.attributes.rgb_color;
-                  if (variables.ulm_card_light_enable_color == true){
-                    if(variables.ulm_card_light_force_background_color == true || hass.themes.darkMode){
-                      if (color){
-                        return 'rgba(' + color + ',0.1)'
+                  if (variables.ulm_card_light_enable_color) {
+                    if (variables.ulm_card_light_force_background_color || hass.themes.darkMode) {
+                      if (color) {
+                        return 'rgba(' + color + ',var(--opacity-bg))';
```

Comment:
Breaks the card when `ulm_card_light_force_background_color == true` and using light-mode.
`var(--opacity-bg)` = `1` in light moder
<img width="528" alt="image" src="https://user-images.githubusercontent.com/68892092/224012918-8721f045-017d-4e8b-859c-30d7a01688f4.png">

Reply:
Have updated this. Having an issue now with the Colour around the 3 item list - will look into this.
![image](https://user-images.githubusercontent.com/77846694/224017723-b656c6e5-3b6b-4e6f-8602-3212ac796edb.png)

Reply:
Just add a style to the list template to remove the background
```yaml
styles:
  card:
    - background: "none"
```

Reply:
Perfect, thanks for the tip.